### PR TITLE
chore(github-action): Avoid builds for pushes to helm chart and docs files

### PR DESCRIPTION
**Pull request by niladrih**
This condition in the .github/workflows/build.yml file will ignore pushes made to `docs/` and `deploy/helm/` directories and avoid building the images.

Signed-off-by: Niladri Halder <niladri.halder@mayadata.io>

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -13,7 +13,11 @@
 # limitations under the License.
 name: build
 
-on: ['push']
+on:
+  push:
+    paths-ignore:
+    - 'docs/**'
+    - 'deploy/helm/**'
```
Comment (akhilerm) [inline .github/workflows/build.yml:20]: The same change needs to be done for pull request workflow also. Can you also add, changelog directory to `paths-ignore`

**Review by akhilerm — COMMENTED** [review]
Given some commets

**Comment by niladrih** [comment]
@akhilerm -- Added `changelogs/`, changed `deploy/helm/**` to `deploy/`, `docs/` remains. Made these changes to both build.yml and pull_request.yml files.

**Comment by niladrih** [comment]
@akhilerm -- Changed `deploy/` back to `deploy/helm` because of the reasons you cited -- Changes to the deploy files affect integration testing for the components.

**Review by akhilerm — APPROVED** [review]
LGTM

**Comment by niladrih** [comment]
Verified this.

**Comment by niladrih** [comment]
Related PRs:
https://github.com/openebs/cstor-operators/pull/271
https://github.com/openebs/dynamic-localpv-provisioner/pull/41
https://github.com/openebs/zfs-localpv/pull/306
https://github.com/openebs/lvm-localpv/pull/42
https://github.com/openebs/jiva-operator/pull/66
https://github.com/openebs/maya/pull/1797
https://github.com/openebs/cstor-csi/pull/151
https://github.com/openebs/jiva/pull/350
https://github.com/openebs/libcstor/pull/85
https://github.com/openebs/velero-plugin/pull/151
https://github.com/openebs/m-exporter/pull/14

**Pull request merged**